src/app/config/fallback_dirs.inc

```diff
@@ -1,6 +1,2 @@
 // Temporary list of fallback directory mirrors to prevent ator-protocol connecting
 // to the original Tor network during development
```

Comment:
I would remove these lines as well, since we do not have temporary list anymore

Reply:
It might make sense to have fallback dirs in place after we have some stable long-running relays with dir caches